### ISSUE TITLE
Caused by IllegalArgumentException: No module found with name [bootstrap]

### ISSUE THREAD

**Issue by orlandobcrra** [joergrech/KickstartWithBootstrap#15]
Con la nueva versión del plugin, hay que mantenerlo instalado. Como hago para no depender del plugin y que la aplicación funcione? Gracias.

| Running Grails application
| Error 2012-10-23 10:26:43,604 [localhost-startStop-1] ERROR resource.ResourceProcessor  - Unable to load resources
Message: Cannot create resource /less/bootstrap/bootstrap.less, is not a supported type
    Line | Method
->>  107 | newResourceFromArgs     in org.grails.plugin.resource.ResourceModule

---

|     64 | doCall                  in org.grails.plugin.resource.ResourceModule$_closure1
|     58 | <init> . . . . . . . .  in org.grails.plugin.resource.ResourceModule
|    681 | defineModuleFromBuilder in org.grails.plugin.resource.ResourceProcessor
|    802 | doCall . . . . . . . .  in org.grails.plugin.resource.ResourceProcessor$_loadModules_closure19
|   1075 | reloadAll               in org.grails.plugin.resource.ResourceProcessor
|    172 | doCall . . . . . . . .  in ResourcesGrailsPlugin$_closure3
|    303 | innerRun                in java.util.concurrent.FutureTask$Sync
|    138 | run . . . . . . . . . . in java.util.concurrent.FutureTask
|    886 | runTask                 in java.util.concurrent.ThreadPoolExecutor$Worker
|    908 | run . . . . . . . . . . in     ''
^    662 | run                     in java.lang.Thread
| Server running. Browse to http://localhost:8080/GrailsApplication6
| Error 2012-10-23 10:26:46,688 [http-bio-8080-exec-2] ERROR [/GrailsApplication6].[grails-errorhandler]  - Servlet.service() for servlet grails-errorhandler threw exception
Message: Error applying layout : kickstart
   Line | Method
->> 886 | runTask in \grails-app\views\layouts\kickstart.gsp

---

|   908 | run     in     ''
^   662 | run . . in     ''

Caused by GroovyPagesException: Error processing GroovyPageView: Error executing tag <r:layoutResources>: No module found with name [bootstrap]
->> 464 | runTask in \grails-app\views\layouts\kickstart.gsp

---

Caused by GrailsTagException: Error executing tag <r:layoutResources>: No module found with name [bootstrap]
->>  17 | doCall  in C:/Users/globalr/Documents/NetBeansProjects/GrailsApplication6/grails-app/views/layouts/kickstart.gsp

---

Caused by IllegalArgumentException: No module found with name [bootstrap]
->> 1132 | getAllModuleNamesRequired in org.grails.plugin.resource.ResourceProcessor

---

|   398 | doCall  in org.grails.plugin.resource.ResourceTagLib$_closure6
|    39 | doCall  in C__Users_globalr_Documents_NetBeansProjects_GrailsApplication6_grails_app_views_layouts_kickstart_gsp$_run_closure2
|    44 | run     in C__Users_globalr_Documents_NetBeansProjects_GrailsApplication6_grails_app_views_layouts_kickstart_gsp
|   886 | runTask in java.util.concurrent.ThreadPoolExecutor$Worker
|   908 | run     in     ''
^   662 | run . . in java.lang.Thread
| Error 2012-10-23 10:26:46,826 [http-bio-8080-exec-2] ERROR [Tomcat].[localhost]  - Exception Processing ErrorPage[errorCode=404, location=/grails-errorhandler]
Message: Error applying layout : kickstart
   Line | Method
->> 886 | runTask in \grails-app\views\layouts\kickstart.gsp

---

|   908 | run     in     ''
^   662 | run . . in     ''

Caused by GroovyPagesException: Error processing GroovyPageView: Error executing tag <r:layoutResources>: No module found with name [bootstrap]
->> 464 | runTask in \grails-app\views\layouts\kickstart.gsp

---

Caused by GrailsTagException: Error executing tag <r:layoutResources>: No module found with name [bootstrap]
->>  17 | doCall  in C:/Users/globalr/Documents/NetBeansProjects/GrailsApplication6/grails-app/views/layouts/kickstart.gsp

---

Caused by IllegalArgumentException: No module found with name [bootstrap]
->> 1132 | getAllModuleNamesRequired in org.grails.plugin.resource.ResourceProcessor

---

|   398 | doCall  in org.grails.plugin.resource.ResourceTagLib$_closure6
|    39 | doCall  in C__Users_globalr_Documents_NetBeansProjects_GrailsApplication6_grails_app_views_layouts_kickstart_gsp$_run_closure2
|    44 | run     in C__Users_globalr_Documents_NetBeansProjects_GrailsApplication6_grails_app_views_layouts_kickstart_gsp
|   886 | runTask in java.util.concurrent.ThreadPoolExecutor$Worker
|   908 | run     in     ''
^   662 | run . . in java.lang.Thread

**Comment by joergrech** [comment]
Hi, as I do not speak Spanish or Portugese (sorry) so I try to answer in English. 

The problem is that the plugin **lesscss-resources** is not installed in your project. Therefore, the resources plugin does not know what to do with less files. 

However, the lesscss-resource plugin should be installed if you use Kickstart version 0.8.0. Did you used a 0.7 version? You can also check the BuildConfig.groovy file within the Kickstart plugin folder - it should contain an entry **runtime ":lesscss-resources:1.3.0.3"** in the plugin part.

**Comment by jeltok** [comment]
Hi Jörg

I have experienced the same problem with "No module found with name [bootstrap]" error while using your (excellent) plugin version 0.8.9.  lesscss-resources:1.3.0.3 was appearing under plugins just right.

After some research I figured out that I have defined my own grails.resources.modules = { ... } in Config.groovy

As soon as I moved my modules to ApplicationResources.groovy file, and deleted grails.resources.modules from Config.groovy, the system started to work fine again.

Looks like there is some kind of collision for grails.resources.modules configuration parameter.

Thank you.
